stock_location_route_description/models/stock_location_route.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class StockLocationRoute(models.Model):
+    _inherit = "stock.location.route"
+
+    description = fields.Char()
+
+    def name_get(self):
+        result = []
+        if self.env.context.get("show_description", False):
```

Comment:
what happens if there is no show_description context?

Reply:
I'm working on it, I forgot it